lib/ProductOpener/Units.pm

```diff
@@ -269,14 +276,14 @@ sub normalize_quantity ($quantity) {
 	# 6 bricks de 1 l
 	# 10 unités, 170 g
 	# 4 bouteilles en verre de 20cl
-	if ($quantity =~ /(\d+)(\s(\p{Letter}| )+)?(\s)?( de | of |x|\*)(\s)?((\d+)(\.|,)?(\d+)?)(\s)?($units)/i) {
+	if ($quantity =~ /(\d+)(\s(\p{Letter}| )+)?(\s)?( de | of |x|\*)(\s)?((\d+)(\.|,)?(\d+)?)(\s)?($units)\s*\b/i) {
```

Comment:
why this `\s*\b` it seems a bit strange (spaces followed by a boundary ?). Why not just `\b` ?

Should be good to explain it in the comment above.

Reply:
Just \b should work indeed, I'll fix it.